src/test/java/domain/dao/impl/FileFlightDaoTests.java

```diff
@@ -1,4 +1,91 @@
 package domain.dao.impl;
 
+import domain.entity.FlightEntity;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Consider avoiding wildcard imports (*). Please replace them with explicit imports to improve readability and maintain consistency with our coding standards.